src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -108,19 +104,7 @@ class AuthScreens extends React.Component {
         });
 
         // Fetch some data we need on initialization
-        NameValuePair.get(CONST.NVP.PRIORITY_MODE, ONYXKEYS.NVP_PRIORITY_MODE, 'default');
-        NameValuePair.get(CONST.NVP.IS_FIRST_TIME_NEW_EXPENSIFY_USER, ONYXKEYS.NVP_IS_FIRST_TIME_NEW_EXPENSIFY_USER, true);
-        App.getLocale();
-        PersonalDetails.fetchPersonalDetails();
-        User.getUserDetails();
-        User.getBetas();
-        User.getDomainInfo();
-        PersonalDetails.fetchLocalCurrency();
-        Report.fetchAllReports(true, true);
-        GeoLocation.fetchCountryCodeByRequestIP();
-        UnreadIndicatorUpdater.listenForReportChanges();
-        BankAccounts.fetchFreePlanVerifiedBankAccount();
-        BankAccounts.fetchUserWallet();
```

Comment:
One weird thing is that we are also getting the policy list here:

https://github.com/Expensify/App/blob/d3e1840a3bfa61599062442f7aae3e007e3893cf/src/libs/Navigation/AppNavigator/AuthScreens.js#L112-L117

We do it in a specific way because we might be landing on AuthScreens via the `/transition` route and we need to create our initial policy before we sync all the policies.

The best suggestion I can come up with for now is to add something like `shouldSyncPolicyList = true` argument then do:

```js
App.getAppData(false);
```